README.md

```diff
@@ -94,6 +96,7 @@ module "ipa1" {
 | nessus\_hostname\_key | The SSM Parameter Store key whose corresponding value contains the hostname of the CDM Tenable Nessus server to which the Nessus Agent should link (e.g. /cdm/nessus/hostname). | `string` | n/a | yes |
 | nessus\_key\_key | The SSM Parameter Store key whose corresponding value contains the secret key that the Nessus Agent should use when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/key). | `string` | n/a | yes |
 | nessus\_port\_key | The SSM Parameter Store key whose corresponding value contains the port to which the Nessus Agent should connect when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/port). | `string` | n/a | yes |
+| netbios\_name | The NETBIOS name to be used by the server (e.g. EXAMPLE).  Note that NETBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes. | `string` | n/a | yes |
```

Comment:
Given the limit should we add something like in https://github.com/cisagov/ssm-read-role-tf-module/pull/19 (and related) to truncate down to 15 characters.?

Reply:
I think the validation code I added in `variables.tf` is more than sufficient.  In the other case the role names are automatically generated, so it makes more sense to truncate without consulting the user.  In this case it makes more sense to reject the overly long name and make the user decide what he or she wants to do.